packages/functional-tests/scripts/start-services.sh

```diff
@@ -9,6 +9,7 @@ cd "$DIR/../../../"
 
 mkdir -p ~/.pm2/logs
 mkdir -p artifacts/tests
+chmod +x node_modules/@nestjs/cli/bin/nest.js
```

Comment:
Weird. Do we need to file this upstream?

Reply:
Actually, this got me curious enough to open up ye old laptop.

```sh
npm init -y && npm i @nestjs/cli -D

npx nest --version
# 10.3.2

ls -lash node_modules/.bin/ | grep "nest"
# 0 lrwxr-xr-x    1 pdehaan  wheel    26B 23 Apr 23:30 nest -> ../@nestjs/cli/bin/nest.js

ls -lash node_modules/@nestjs/cli/bin/nest.js
# 8 -rwxr-xr-x  1 pdehaan  wheel   923B 23 Apr 23:30 node_modules/@nestjs/cli/bin/nest.js
```

Now I'm confused.

Reply:
It's due to an update in place on an existing install or with cached node_modules.
https://github.com/yarnpkg/berry/issues/5344